x-pack/plugins/infra/public/pages/metrics/hosts/components/hosts_table_columns.tsx

```diff
@@ -10,28 +10,29 @@ import React from 'react';
 import { i18n } from '@kbn/i18n';
 import { EuiText } from '@elastic/eui';
 import { scaleUpPercentage } from '../../../../components/infrastructure_node_metrics_tables/shared/hooks';
-import type { SnapshotNodeMetric } from '../../../../../common/http_api/snapshot_api';
 import { NumberCell } from '../../../../components/infrastructure_node_metrics_tables/shared/components';
 
-interface HostNodeRow extends HostMetics {
+export interface HostNodeRow extends HostMetics {
   os?: string | null | undefined;
   servicesOnHost?: number | null | undefined;
+  name: string;
 }
 
 export interface HostMetics {
```

Comment:
I think we could achieve the same results, without changing the types here. I played with it and `EuiInMemoryTable` complains that `item` type is not compatible because `name` is now declared as a mandatory field. If we make it optional, it should work without these changes here and on `HostsTableColumns` - correct me if I'm wrong or missing something.